clis/ph-cli/src/commands/list.ts

```diff
@@ -0,0 +1,49 @@
+import { Command } from "commander";
+import path from "node:path";
+import { CommandActionType } from "../types.js";
+import { getProjectInfo } from "../utils.js";
+
+export const list: CommandActionType<
+  [
+    {
+      debug?: boolean;
+    },
+  ]
+> = async (options) => {
+  if (options.debug) {
+    console.log(">>> command arguments", { options });
+  }
+
+  const projectInfo = getProjectInfo(options.debug);
+
+  if (options.debug) {
+    console.log("\n>>> projectInfo", projectInfo);
+  }
+
+  try {
+    const phConfig = (await import(
+      path.join(projectInfo.path, "powerhouse.config.json")
+    )) as { packages: { packageName: string }[] };
```

Comment:
Let's use `as Pick<PowerhouseConfig, "packages:>`.
This way if we change that type in the future we'll get a ts error here if packages structure changes.